eng/common/TestResources/clouds/AzureCloud.json

```diff
@@ -2,6 +2,7 @@
     "azureAuthorityHost": "https://login.microsoftonline.com/",
     "cognitiveServicesEndpointSuffix": ".cognitiveservices.azure.com",
     "communicationServicesEndpointSuffix": ".communication.azure.com",
+    "cosmosEndpointSuffix": "cosmos.azure.com",
```

Comment:
We disallow changes to these files as they get synced from another repository. I've created a PR for this [here](https://github.com/Azure/azure-sdk-tools/pull/4622).